src/twp-editor-plugin/files/input-rules-templates/pm-plugins/input-rules.ts

```diff
@@ -0,0 +1,22 @@
+import { InputRule } from 'prosemirror-inputrules';
+import { Plugin } from 'prosemirror-state';
+import { Schema } from 'prosemirror-model';
+import { createInputRule, instrumentedInputRule } from '../../../utils/input-rules';
+
+function inputRulesPlugin(schema: Schema): Plugin | undefined {
+  const rules: InputRule[] = [];
+
+  /**
+   * Bind autoformatting rules to Prosemirror transactions using createInputRule helper:
+   *  const rule = createInputRule(regex, (state, match, start, end) => tr);
+   *  rules.push(rule);
+   */
+
+  if (rules.length !== 0) {
```

Comment:
I noticed this in our codebase too, but why do we use `!== 0` instead of `> 0`? Is it possible/valid for length to be negative?

Reply:
I don't really mind but I guess it's more intuitive for me to read `> 0` for array length (seems like a more common pattern)

But that's just a nit, I'll leave it up to you 🤷‍♂️